src/backtrace/libunwind.rs

```diff
@@ -51,6 +51,16 @@ impl Frame {
         unsafe { uw::_Unwind_GetIP(ctx) as *mut c_void }
     }
 
+    pub fn gr(&self, index: u32) -> Option<usize> {
+        let ctx = match *self {
+            Frame::Raw(ctx) => ctx,
+            Frame::Cloned { .. } => {
+                return None;
+            }
+        };
+        Some(unsafe { uw::_Unwind_GetGR(ctx, index) as usize })
```

Comment:
This is not save: https://github.com/rust-lang/backtrace-rs/pull/291#issuecomment-621207213

> https://github.com/gcc-mirror/gcc/blob/a7201a085cc30f89944931d8fb1d7936f02a169f/libgcc/unwind-dw2.c#L249
>
> ```
> /* This will segfault if the register hasn't been saved.  */
> ```
>
> It seems that `_Unwind_GetGR` is only allowed to be called with saved registers. This means that at least the approach taken in this PR is wrong.